README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
+| Pruning of builds     |       |      |    |
+| Cancelling of builds  |       |      |    |
+| Chaining of builds    |       |      |    |
 | Image Caching         |       |      |    |
+| Max duration / TO     |       |      |    |
+| Parallel vs. serial execution   |       |      |    |
 | ImageStreams support  |       |      |    |
 | Entitlements          |       |      |    |
+| Global Proxy support  |       |      |    |
+| Mirror support        |       |      |    |
+| new-app/build & odo ? |       |      |    |
```

Comment:
this is an interesting one (and I still have a question mark here) .... perhaps first a broad discussion with @sbose78 , @adambkaplan, @bparees, @siamaksade, and myself just the assess the current landscape

Reply:
https://github.com/redhat-developer/build/issues/59